<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QWidget" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QTextBrowser" name="text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>761</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
    </font>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="html">
    <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Monospace','Monospace'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'DejaVu Sans Mono';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="r_Main">
      <property name="text">
       <string>Главная</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="r_Theory">
      <property name="text">
       <string>Теория</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="r_Slovo">
      <property name="text">
       <string>Словарь</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="r_Exercise">
      <property name="text">
       <string>Задачи</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="r_Rules">
      <property name="text">
       <string>Правила</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="r_Test">
      <property name="text">
       <string>Тест</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="r_Statistic">
      <property name="text">
       <string>Статистика</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="r_Names">
      <property name="text">
       <string>Имена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Do">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>560</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
